### PR TITLE
Remove redundant dependency, added missing.

### PR DESCRIPTION
**Pull request by eschlange**
[#98948048]

### CODE REVIEW

**Comment by eschlange** [comment]
@ericcf @cbrenner04  I removed the redundant dependency and added a missing local notifications plug-in. I created a clean slate and successfully ran an Android build. Would you please try building again to see if you receive any errors?

File: package.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
     "angular-resource": "^1.4.0",
     "angular-route": "^1.4.0",
     "cordova": "^5.0.0",
-    "moment": "^2.10.2"
+    "moment": "^2.10.2",
+    "cordova-plugin-local-notifications": "^1.0.0"
```
Comment (ericcf) [inline package.json]: As I mentioned, I recommend that you remove `/plugins` from `.gitignore` and add this plugin there instead.

**Comment by eschlange** [comment]
@ericcf The file count is huger after my most recent update. The only files that should concern your review are: the .gitignore file and the package.json file. Thanks for your review!

**Comment by ericcf** [comment]
:hurtrealbad:

**Pull request merged**